LayoutTest.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was generated by Xcode when I opened the project. From what I understand it is not essential, but should be checked in or added to `.gitignore` - common practice is to check it in,